api/v1beta1/foundationdbcluster_types.go

```diff
@@ -2775,6 +2775,16 @@ type LabelConfig struct {
 	// resources it creates.
 	ResourceLabels map[string]string `json:"resourceLabels,omitempty"`
 
+	// ProcessGroupIDLabels provides the labels that we use for the instance ID
+	// field. The first label will be used by the operator when filtering
+	// resources.
+	ProcessGroupIDLabels []string `json:"processGroupIDLabels,omitempty"`
```

Comment:
Since we are using only using one label for the filtering, what's the purpose of having a `[]string` ?

Reply:
All labels will be added to the resource and this allows the user to move from one set of labels to another set: https://github.com/brownleej/fdb-kubernetes-operator/blob/label-prefix/docs/manual/customization.md#per-resource-labels